README.md

```diff
@@ -0,0 +1,40 @@
+[![logo](./logo.jpg)](https://inforit.nl)
+
+# dev-deployment-cleanup docker image
+
+This container image can be used to automatically drop mongoDB databases en clean-up rabbitmq resources like an exchange, its bindings and destion queues.
+
+## Usage
+
+## environment variables
+
+These environment variables MUST be set! Purpose of this container image is to automatically delete resources after each feature(dev) deployment is deleted from k8s. Database/exchanges names will be different on every run.
```

Comment:
This sentence doesn't read well.

Reply:
What about 

`These environment variables are mandatory and must be set before running this container.` 

?

Reply:
The second one is more formal but there is nothing wrong with the grammar of the first actually :)